src/modules/BattleSimulatorModule/Season.js

```diff
@@ -70,7 +70,9 @@ class Season {
         }
 
         $gridWrapper.find('.matchRating').remove()
-        $gridWrapper.append(`<span class="matchRating ${result.scoreClass}">${I18n.nRounding(100*result.win, 2, -1)}%</span>`)
+        let points = parseInt($opponentData.find('.slot_victory_points :nth-child(1)').text().replace(/[^0-9]/gi, ''))
+        let expected = result.win*points-(1-result.win)*points
```

Comment:
This isn't how Mojo works. The points you lose on loss is 40 - the points you would have won.

Reply:
So this would be the correct formula?

`let expected = result.win*points-(1-result.win)*(40-points)`